packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -1,45 +1,41 @@
-    "complianceBannerTranscriptionSaving": "Transcription is being saved.",
-    "complianceBannerTranscriptionStopped": "Transcription has stopped.",
-    "configurationPageTitle": "Start a call",
-    "defaultPlaceHolder": "Select an option",
-    "failedToJoinCallDueToNoNetworkMoreDetails": "Call was disconnected due to a network issue. Check your connection and join again.",
-    "failedToJoinCallDueToNoNetworkTitle": "Call disconnected",
-    "failedToJoinTeamsMeetingReasonAccessDeniedMoreDetails": "You were not granted entry in the call. If this was a mistake, re-join the call.",
-    "failedToJoinTeamsMeetingReasonAccessDeniedTitle": "Dismissed from lobby",
-    "learnMore": "Learn more",
-    "leftCallMoreDetails": "If this was a mistake, re-join the call.",
-    "leftCallTitle": "You left the call",
-    "lobbyScreenConnectingToCallTitle": "Joining call",
-    "lobbyScreenWaitingToBeAdmittedTitle": "Waiting to be admitted",
-    "microphonePermissionDenied": "Your browser is blocking access to your microphone",
-    "mutedMessage": "You're muted",
-    "networkReconnectMoreDetails": "Looks like something went wrong. We're trying to get back into the call.",
-    "networkReconnectTitle": "Hold on",
-    "privacyPolicy": "Privacy policy",
-    "rejoinCallButtonLabel": "Re-join call",
-    "removedFromCallMoreDetails": "Another participant removed you from the call.",
-    "removedFromCallTitle": "You were removed",
-    "soundLabel": "Sound",
-    "startCallButtonLabel": "Start call",
-    "microphoneToggleInLobbyNotAllowed": "Cannot mute or unmute while in lobby."
-  },
-  "chat": {
-    "chatListHeader": "In this chat"
-  }
+  "cameraLabel": "Camera",
+  "cameraPermissionDenied": "Your browser is blocking access to your camera",
+  "cameraTurnedOff": "Your camera is turned off",
+  "close": "Close",
+  "complianceBannerNowOnlyRecording": "You are now only recording this meeting.",
+  "complianceBannerNowOnlyTranscription": "You are now only transcribing this meeting.",
+  "complianceBannerRecordingAndTranscriptionSaved": "Recording and transcription are being saved.",
+  "complianceBannerRecordingAndTranscriptionStarted": "Recording and transcription have started.",
+  "complianceBannerRecordingAndTranscriptionStopped": "Recording and transcription have stopped.",
+  "complianceBannerRecordingSaving": "Recording is being saved.",
+  "complianceBannerRecordingStarted": "Recording has started.",
+  "complianceBannerRecordingStopped": "Recording has stopped.",
+  "complianceBannerTrancriptionStarted": "Transcription has started.",
+  "complianceBannerTranscriptionConsent": "By joining, you are giving consent for this meeting to be transcribed.",
+  "complianceBannerTranscriptionSaving": "Transcription is being saved.",
+  "complianceBannerTranscriptionStopped": "Transcription has stopped.",
+  "configurationPageTitle": "Start a call",
```

Comment:
what about a config title that is different for meeting for example?
we will have to specify Call or Meeting in the name?
Wouldn't it be better to keep Chat and Call objects and have a common/shared one for all that is common or shared between composites?
Just wondering

Reply:
Yeah, I'm not sure about the right path here either.

This PoC showed, more than anything else, that there isn't a quick answer to be done pre-GA.